subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -303,7 +303,7 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 		adv_iso = ull_adv_iso_by_stream_get(stream_handle);
 		lll_iso = &adv_iso->lll;
 
-		role = 0U; /* FIXME: Set role from LLL struct */
+		role = 0U; /* Unused as broadcast has no ISOAL sink created */
```

Comment:
It's non obvious what this means IMO. Why is `role` only applied for the ISOAL sink for broadcast? It seems to be set and used for either direction for unicast